### ISSUE TITLE
abnormal cerebral blood flow rate

### ISSUE THREAD

**Issue by slaulederkind** [obophenotype/mammalian-phenotype-ontology#3306]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1. abnormal cerebral blood flow rate
2. increased cerebral blood flow rate
3. decreased cerebral blood flow rate

>2. Synonyms (e.g., absent spleen)

1. aberrant cerebral blood flow rate
2. elevated cerebral blood flow rate
3. reduced cerebral blood flow rate

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. aberrant rate of blood flow in the blood vessels supplying the cerebrum
2. elevated rate of blood flow in the blood vessels supplying the cerebrum
3. reduced rate of blood flow in the blood vessels supplying the cerebrum
PMID:25397684

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1. abnormal blood flow velocity (MP:0000233)
2. abnormal cerebral blood flow rate (new term)
3. abnormal cerebral blood flow rate (new term)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-9196-0135
--

**Comment by anna-anagnostop** [comment]
CBF is defined as the volume of arterial blood delivered to a unit mass of brain tissue per unit of time. PMID: 29693564

**Comment by sbello** [comment]
unclear if cerebrum is being used as a synonym for the brain in general, suggest creating a generic brain blood flow rate term and then create a specific cerebrum as child

**Comment by slaulederkind** [comment]
I need specific cerebrum terms for blood flow in the cerebral hemispheres.  Making "brain" parent terms might save you more work later if the "brain" terms would be needed later for some other curation.  Thanks.

**Comment by sbello** [comment]
@slaulederkind that is our plan (create both) since some existing MGI annotations are unclear

**Comment by anna-anagnostop** [comment]
Added new terms:

abnormal brain blood flow rate MP:0031179
decreased brain blood flow rate MP:0031180
increased brain blood flow rate MP:0031181
abnormal cerebral blood flow rate MP:0031182
decreased cerebral blood flow rate MP:0031183
increased cerebral blood flow rate MP:0031184

Still need to decide on what EQs to use; Also, check EQs for abnormal coronary flow rate and abnormal renal plasma flow rate terms

**Comment by anna-anagnostop** [comment]
For decreased terms use abnormallyDecreasedRateOfBiologicalProcessInLocation.yaml:

'has_part' some ('decreased rate' and ('inheres_in' some (%s and ('occurs_in' some %s))) and ('qualifier' some 'abnormal'))

**Comment by anna-anagnostop** [comment]
Added EQs for abnormal coronary flow rate and abnormal renal plasma flow rate terms

Also, amended EQs for abnormal coronary circulation MP:0006111 and abnormal pulmonary circulation MP:0002295 using:

'has_part' some ('process quality' and ('inheres_in' some (%s and ('occurs_in' some %s))) and  ('has_modifier' some 'abnormal'))"
